docs/README.md

```diff
@@ -138,7 +138,9 @@ The cli looks as follows:
 
 ```bash
 ➜ pixi -h
-Usage: pixi [COMMAND]
+A package management and workflow tool
+
+Usage: pixi.exe [COMMAND]
```

Comment:
`pixi.exe` is only used on Windows.

Reply:
Its autogenerated by `clap` so it will be different per OS. But I removed the exe from the readme.